packages/fxa-settings/src/components/InputPassword/index.tsx

```diff
@@ -55,7 +55,7 @@ export const InputPassword = ({
       <button
         type="button"
         data-testid={formatDataTestId('visibility-toggle')}
-        className={`w-5 px-3 py-2 text-grey-600 focus:text-blue-500 box-content ${
+        className={`w-5 px-3 py-2 text-grey-600 focus:outline-none box-content ${
```

Comment:
@karinabk This definitely does what the ticket asked ("make them consistent") since I'm not seeing a focus state on the sign up page password input, but it's actually slightly better for accessibility if we _do_ allow a focus state to show. I think if we let the default focus state show, the fix for the ticket will be satisfied since it will no longer be blue (which is very different inconsistent from the sign up page) but will also be better for a11y.

I notice if we simply remove the `focus:text-blue-500` class, the left side of the default focus outline is not shown:
<img width="542" alt="image" src="https://user-images.githubusercontent.com/13018240/113315794-45b24d80-92d3-11eb-9fe7-9cd2e05a3975.png">

It would be ideal if we could get it to look like this, I think there's one or two different ways to make this happen:
<img width="581" alt="image" src="https://user-images.githubusercontent.com/13018240/113316187-acd00200-92d3-11eb-891c-ea79782a99bb.png">

Let me know if you need help/clarification! 🙂

Reply:
I see. I didn't remove it because on the 'Sign-in' page there is no outline, while on the 'Change password' page the default outline is too noticeable. However, I agree that with outline is better.
![Screenshot from 2021-04-01 22-10-46](https://user-images.githubusercontent.com/35498147/113322924-2220ee80-9337-11eb-8b58-95ff82091243.png)

Should we leave the default outline anyway?
I also noticed that in the Recovery key step 1 (when the user needs to enter the password) eye icon is also not consistent. Do I need to fix it in this PR?

Reply:
Totally makes sense, you did exactly what the ticket asked!

Out of curiosity, what OS / browser are you using? I do think we should leave the default outline for now - we can likely address improving how it looks across `fxa-settings` in another issue because I imagine if it looks like that here, the focus state likely looks like that in other places on this page. Let's leave it for now.

It looks like the same component, `InputPassword`, is used across the board there (including in the Recovery key step 1), so if we fix it here, it should be fixed everywhere we use this component! Good thinking though👍

Reply:
I'm using Chrome Version 89.0.4389.90 (Official Build) on Ubuntu.
I forgot that it can be the same component :sweat_smile: In this case, why we cannot use the same component on the Sign-in page?
P.S I updated PR

Reply:
Ah that makes sense! Since a very large chunk of our users use Firefox, that's the primary one we look at. I checked out your latest push, and I'm still not seeing the outline on the left-hand side:

<img width="299" alt="image" src="https://user-images.githubusercontent.com/13018240/113437976-45868080-93ad-11eb-83de-92c605b2783e.png">

If we change the width back (I'm guessing it's at the size it is to create a larger clickable space for users using a mouse, but also users on a phone tapping), remove the negative margin on the button, and remove the adjacent span's `relative` class, that seems to work! Double check me though, I'm not sure why that `relative` class would need to be on there in the first place.

>  In this case, why we cannot use the same component on the Sign-in page?

That is a great question! We recently revamped the Settings page and it uses React. The sign-in/up page is still using Backbone which the Settings page also used until recently; I'm not sure when we'll get to it but we hope to eventually extend React over into the login flow, and at that point, they'll use the same component. This is totally not necessary to check out, but we have some related ADRs [here](https://github.com/mozilla/fxa/blob/main/docs/adr/0010-transition-fxa-from-backbone-to-react.md) and [here](https://github.com/mozilla/fxa/blob/main/docs/adr/0011-create-new-react-app-for-settings-redesign.md) about that conversion.

Reply:
Thank you for your hints! I changed PR and squashed commits